nafas/__main__.py

```diff
@@ -1,11 +1,18 @@
 # -*- coding: utf-8 -*-
 """nafas main."""
 
+import sys
 from .functions import *
+from art import tprint
+from .params import NAFAS_VERSION
 
 if __name__ == "__main__":
+    args = sys.argv
+    tprint("Nafas")
+    tprint("v" + str(NAFAS_VERSION))
     description_print()
-    input_data = get_input_standard()
-    filtered_data = input_filter(input_data)
-    program_data = get_program_dict(filtered_data)
-    run(program_data)
\ No newline at end of file
+    if len(args) < 1:
+        input_data = get_input_standard()
+        filtered_data = input_filter(input_data)
+        program_data = get_program_dict(filtered_data)
+        run(program_data)
```

Comment:
A new line here would be proper.

Reply:
Thanks ;-)
Solved in https://github.com/sepandhaghighi/nafas/pull/9/commits/bf512418224172d4281fc3594cd569a567f33039